File: app/public_html/script/functions/samples/21.py
r = 1
def loop():
    global r
    while isSafeHere() and not moveTo(r,r):
        pass
    while isSafeHere() and not moveTo(25-r,r):
        pass
    while isSafeHere() and not moveTo(25-r,25-r):
        pass
    while isSafeHere() and not moveTo(r,25-r):
        pass
    while getDist(12.5,12.5) >= getSafeRadius() - 2:
        moveTo(12.5,12.5)
    r = 12.5 - getSafeRadius()/2
 

File: app/public_html/script/functions/samples/21.c
float r = 1;
loop(){
    while(isSafeHere() && !moveTo(r,r));
    while(isSafeHere() && !moveTo(25-r,r));
    while(isSafeHere() && !moveTo(25-r,25-r));
    while(isSafeHere() && !moveTo(r,25-r));
    while (getDist(12.5,12.5) >= getSafeRadius() - 2)
        moveTo(12.5,12.5);
    r = 12.5 - getSafeRadius()/2;
}